index.html

```diff
@@ -26,25 +26,30 @@
 </head>
 
 <body>
+  <header>
+    <nav class="navbar">
+      <a href="#homepage-section" class="nav-branding">My Logo</a>
+      <ul class="nav-menu">
+        <li class="nav-item">
+          <a href="#homepage-section" class="nav-link">Portfolio</a>
```

Comment:
It's really nice that all links are going to the appropriate sections. 👍 However, what do you think about changing the `href` attribute of `Portfolio` from `#homepage-section` to the section that contains the projects that you're showcasing?
That way, it'll be easier for anyone viewing your portfolio on small screens to quickly see your works. I hope this makes sense. 😁